Makefile

```diff
@@ -1,50 +1,62 @@
 GCR_URL = us.gcr.io/vcm-ml
 DOCKERFILE ?= docker/Dockerfile
-COMPILED_TAG_NAME ?= default
+COMPILED_TAG_NAME ?= latest
 ENVIRONMENT_TAG_NAME ?= latest
 COMPILE_OPTION ?=
 COMPILE_TARGET ?= fv3gfs-compiled
 ENVIRONMENT_TARGET ?= fv3gfs-environment
 COMPILED_IMAGE ?= $(GCR_URL)/$(COMPILE_TARGET):$(COMPILED_TAG_NAME)
+SERIALIZE_IMAGE ?= $(GCR_URL)/$(COMPILE_TARGET):$(COMPILED_TAG_NAME)-serialize
 ENVIRONMENT_IMAGE=$(GCR_URL)/$(ENVIRONMENT_TARGET):$(ENVIRONMENT_TAG_NAME)
```

Comment:
So before using the TARGET to for the IMAGE worked because the serialize targets matched the default target names and were in different files. Will this setup generate
us.gcr.io/vcm-ml/fv3gfs-environment:latest
us.gcr.io/vcm-ml/fv3gfs-environment-serialbox:serialize
us.gcr.io/vcm-ml/fv3gfs-compiled:latest
us.gcr.io/vcm-ml/fv3gfs-compiled-serialbox:serialize
? Or am I missing something. We might not want to have both the image name and the tag indicate serialization, though it is just a name :).

Reply:
But I can just use 	ENVIRONMENT_TARGET=fv3gfs-environment-serialbox \
	ENVIRONMENT_IMAGE=$(GCR_URL)/fv3gfs-environment:serialize, just the reuse of constants doesn't work the same way anymore, that's fine.

Reply:
`fv3gfs-compiled:serialize` is based off `fv3gfs-environment`, not `fv3gfs-environment-serialize`.